components/Snackbar/src/ColorThemer/MDCSnackbarColorThemer.m

```diff
@@ -18,10 +18,18 @@
 
 @implementation MDCSnackbarColorThemer
 
-+ (void)applyColorScheme:(nonnull id<MDCColorScheme>)colorScheme
-    toSnackbarMessageView:(nonnull MDCSnackbarMessageView *)snackbarMessageView {
-  // TODO(yar): Currently a stub. Will need to update the color setting once we have concluded
-  // the proper color theming and scheme.
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme {
+  MDCSnackbarManager.snackbarMessageViewBackgroundColor =
+      [colorScheme.onSurfaceColor colorWithAlphaComponent:0.8f];
```

Comment:
Will confirm. I looked up the params and also commented there, but afaik snackbar is considered a "dark theme" component and hence uses the opposite theming; light content on dark surface rather than vice versa.